sdk/tables/Azure.Data.Tables/api/Azure.Data.Tables.netstandard2.0.cs

```diff
@@ -186,3 +190,147 @@ public partial class TableResponseProperties
         public string TableName { get { throw null; } }
     }
 }
+namespace Azure.Data.Tables.Sas
+{
+    [System.FlagsAttribute]
+    public enum AccountSasServices
+    {
+        Tables = 8,
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct SasIPRange : System.IEquatable<Azure.Data.Tables.Sas.SasIPRange>
```

Comment:
I wonder if it's worth pulling this into `Azure.Core`.  It's too late for Storage, but it probably won't be the last `IPRange` we need.

Reply:
Logged #11775 to consider this.